components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -435,6 +435,7 @@ def pickFirstSignature(condition, filterLambda):
                     # large enough that we can examine this argument.
                     info = getJSToNativeConversionInfo(
                         type, descriptor, failureCode="break;", isDefinitelyObject=True)
+                    assert not info.needsRooting
```

Comment:
Does this mean we don't support overloading that involves typed arrays? Will this restriction make https://dxr.mozilla.org/servo/rev/b8df502491b880cb942badee4ddbf7ec65b174fb/components/script/dom/webidls/WebGLRenderingContext.webidl#491-505 impossible?